app/controllers/exam_templates_controller.rb

```diff
@@ -73,6 +73,37 @@ def update
     redirect_to action: 'index'
   end
 
+  def create_template_division
+    assignment = Assignment.find(params[:assignment_id])
+    template = assignment.exam_templates.find_by(id: params[:id])
+    division_start = params[:create_division][:start]
+    division_end = params[:create_division][:end]
+    division_label = params[:create_division][:label]
+
+    new_template_division = template.template_divisions.create_with_associations(
```

Comment:
I believe `create_*` saves already. You might want `new_with...` or `initialize_with...` But also I'm not sure why the association is necessary?